### PR TITLE
Possible fix for #12048.

### PR DESCRIPTION
**Pull request by rmoravcik**
Fix for problem reported in #12048.

### CODE REVIEW

File: Marlin/src/module/configuration_store.cpp [hunk 1]
```diff
@@ -620,8 +620,6 @@ void MarlinSettings::postprocess() {
 
       #if ENABLED(Z_TRIPLE_ENDSTOPS)
         EEPROM_WRITE(endstops.z3_endstop_adj);   // 1 float
-      #else
-        EEPROM_WRITE(dummy);
       #endif
```
Comment (thinkyhead) [inline Marlin/src/module/configuration_store.cpp]: We write placeholders for disabled features, and as long as there's parity between the read and the write, all should be well. So the correct solution is to always include z3_endstop_adj in the EEPROM data structure.

**Comment by dahuafschmied** [comment]
@thinkyhead but with enabled dummies you have to delete 
198    #if ENABLED(Z_TRIPLE_ENDSTOPS)
and
200    #endif
to remove the eeprom error.

-> #12112

**Comment by thinkyhead** [comment]
Too slow, @dahuafschmied! I've already applied that correction.

**Comment by boelle** [comment]
@thinkyhead so this one  and #12112 is allready in bugfix 2?

**Comment by thinkyhead** [comment]
That's what the "Merged" status indicator means, @boelle.

**Comment by boelle** [comment]
yeah yeah i know, just have to be sure

just got out of bed and brain is still warming up

but will do an test later and report back in #12048

**Pull request merged**